projects/vdk-core/src/vdk/internal/builtin_plugins/connection/database_managed_connection.py

```diff
@@ -97,7 +97,7 @@ def db_connection_recover_operation(self, recovery_cursor: RecoveryCursor) -> No
             Provides operation details and tooling.
         :return:
         """
-        pass
+        raise recovery_cursor.get_exception()
```

Comment:
Is this a change in behaviour?

Reply:
@murphp15 no, it doesn't change the behavior.
 The error is raised as default later in code, but it is best it to be here so we avoid additional checks afterwards